bylaws/001-mission-and-structure.md

```diff
@@ -28,6 +28,14 @@ If, in the judgement of the PHP FIG, a Project Representative is acting inapprop
 
 A Project Representative may also simultaneously serve as a Core Committee member, PSR Editor or Working Group member but not as a Secretary.
 
+## Conduct
+
+All individuals involved with FIG either in an official capacity (Secretary, Core Committee, Project Representative, Editor, or Working Group member) or informally (such as participating in discussions) are expected to conduct themselves in a professional manner at all times.  All individuals are expected to behave in a manner consistent with the Code Manifesto.  Failure to do so will be considered grounds for expulsion.
```

Comment:
> Failure to do so will be considered grounds for expulsion.

I'm not sure this statement is necessary. The following paragraphs go on to state what can happen if behaviour is unprofessional but this sentence is very blanket. Expulsion from what? It depends. 'Grounds' is also a word I'd steer clear of as that implies there has to be a reason for expulsion votes, and currently, in bylaws, there isn't.